.github/workflows/matplotlib-tests.yml

```diff
@@ -0,0 +1,69 @@
+name: matplotlib-tests
+
+on:
+  pull_request:
+    paths:
+      - tests/visualization_tests/**
+      - .github/workflows/matplotlib-tests.yml
+      - optuna/visualization/**.py
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  matplotlib-tests:
+    runs-on: ubuntu-latest
+
+    # Scheduled Tests are disabled for forks.
+    if: (github.event_name == 'schedule' && github.repository == 'optuna/optuna') || (github.event_name != 'schedule')
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup Python3.8
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+
+    - name: Setup cache
+      uses: actions/cache@v3
+      env:
+        cache-name: test-with-minimum-versions
+      with:
+        path:  ~/.cache/pip
+        key: ${{ runner.os }}-3.8-${{ env.cache-name }}-${{ hashFiles('**/pyproject.toml') }}-v1
+        restore-keys: |
+          ${{ runner.os }}-3.8-${{ env.cache-name }}-${{ hashFiles('**/pyproject.toml') }}
+
+    - name: Install
+      run: |
+        python -m pip install --upgrade pip
+        pip install --progress-bar off -U setuptools
+        # Install minimal dependencies and confirm that `import optuna` is successful.
+        pip install --progress-bar off .
+        python -c 'import optuna'
+        optuna --version
+        pip install --progress-bar off .[test]
+        pip install --progress-bar off .[optional]
+
+    - name: Output installed packages
+      run: |
+        pip freeze --all
+    - name: Output dependency tree
+      run: |
+        pip install pipdeptree
+        pipdeptree
+
+
+    - name: Scheduled tests
+      if:  ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
+      run: |
+        pytest tests -m "not integration"
```

Comment:
By the same as one of the previous my review comments, this seems to be done by another CI job and not related to visualization part.

```suggestion
```